src/api/index.ts

```diff
@@ -8,3 +8,4 @@ export * from './itemLogin';
 export * from './itemFlag';
 export * from './chat';
 export * from './category';
+export * from './keywordSearch';
```

Comment:
Just wondering, would it be better to call it `search`? I don't know if we plan other types of search? I'm mainly talking about the file name (and some other general places, but the name of the hook looks okay)